docker/developers/docker-compose.dashboard.yml

```diff
@@ -0,0 +1,32 @@
+include:
+  - path: ./docker-compose.mysql.yml
+  - path: ./docker-compose.redis.yml
+  - path: ./docker-compose.networks.yml
+
+services:
+  # Runs all the dependent services when the dashboard server is running natively
+  dashboard-services:
+    image: alpine
```

Comment:
Out of curiosity, did we consider `ubuntu-server` for consistency with our Production cluster or does `alpine` make the most sense?

Reply:
In PR 4, you can see the fully contained dashboard server which will indeed use ubuntu to match the version.